hooks/use-iframe-theme-injector.ts

```
@@ -0,0 +1,175 @@
+          console.log("Tweakcn Embed: Embed loaded");
+          setThemeInjectionError(null);
+          break;
+
+        case MESSAGE.PONG:
+          setStatus("connected");
+          setThemeInjectionError(null);
+          postMessage({ type: MESSAGE.CHECK_SHADCN });
+          break;
+
+        case MESSAGE.SHADCN_STATUS:
+          const { supported } = message.payload;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)